cpp/ps0003/Task3.cpp

```diff
@@ -4,18 +4,21 @@ using namespace std;
 
 int main()
 {
+    // prompting user for number 2/2
     float x;
 
     cout << "Enter your number: \n";
     cin >> x;
 
+    // checking if the number is odd or even 3/3
     if (x % 2 == 0)
     {
-    cout << "This number is even \n";
+        cout << "This number is even \n";
     }
     else
     {
-    cout << "This number is odd \n";
+        cout << "This number is odd \n";
     }
     return 0;
 }
+// great work 😎 9/10 ⭐

```

Comment:
- [x] program returns `0` as everything is okay
- [x] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code